### ISSUE TITLE
Feature Request - qualify Fun Flexible Features

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/documentation-wiki#308]
**Is your feature request related to a problem? Please describe.**

FFFs are a little cryptic eg in the 'select named geography' window; they are currently just "Baker."

**Describe what you're trying to accomplish** 

Make things less cryptic without complicating the UI.

**Describe the solution you'd like**

suffix quads with ' Quad' and islands which don't contain 'island' with ' Island'

**Describe alternatives you've considered**

Leave them cryptic or do something involving more structure/complication.

**Additional context**

-

**Priority**

Might be high for some users?

@ArctosDB/geo-group thoughts?

**Comment by campmlc** [comment]
I'd say priority high given this affects our educational tools and broader
impacts.

**Comment by mkoo** [comment]
I was thinking of this issue-- a definition for FFF. Let me deal wtih the MOU revamp this morning and I will start a definition draft both to introduce the new database and conceptual concepts to the AWG and users and as a working model for us to navigate this new space

**Comment by dustymc** [comment]
>priority high

Here's a QnD pass, if you want to clean anything in maybe_new_i or maybe_new_q or just confirm that this looks sane I can update.

And I suppose there's no technical reason to avoid intersections in FFF, so if you want to add other geography columns you can do that too. I suspect that just reduces usability and deserves at least discussion, but technically it's just metadata of a shape and I suppose I don't need to care what that looks like.

```

create table temp_fff as select
geog_auth_rec_id,
higher_geog,
island,
island || ' Island' as maybe_new_i,
quad,
quad || ' Quad' as maybe_new_q
from geog_auth_rec where 
valid_catalog_term_fg=0 and (island is not null or quad is not null);
update temp_fff set maybe_new_i=trim(replace(maybe_new_i, ' Island Island',' Island'))   where maybe_new_i like '%Island Island%';
```
[temp_fff.csv.zip](https://github.com/ArctosDB/arctos/files/9931010/temp_fff.csv.zip)

**Comment by mkoo** [comment]
Most of these will be converted to locality attributes-- in fact I think all should be. So rather than qualify let's take that approach and geo group can discuss and roadmap the migration. 

Can I rename to "Plan for the Future of Features" or somesuch?

EDIT: oops-- sorry just realized it's in the Doc repo-- so should be renamed as "REwrite page to reflect the Plan for the Future of Features"!

**Comment by dustymc** [comment]
>converted to locality attributes-- in fact I think all should be

Yes. (Not actually, they should remain FFFs for searchability, but it's become clear that should also be separate from assertions - for which locality attributes are appropriate and these terms could be used.)

>rename

Please!

> realized it's in the Doc repo

Possibly because I'm lost - I'm just trying to categorize and sort and deal with what can be dealt with, please feel very free to put this (and friends) wherever it needs to be and clarify it however might be useful. Thanks!
